site/about/styles.css

```diff
@@ -170,62 +170,33 @@
 
  /* HISTORY ROW ONTOP */
 
- .box-1 {
+ .story-box {
+     padding: 50px 200px 50px 200px;
      background-color: lightgray;
-     padding: 100px 155px 100px 155px;
-     margin: 20px;
- }
-
- .box-2 {
-     background-color: gray;
-     padding: 100px 100px 100px 100px;
-     margin: 20px;
- }
-
- .box-3 {
-     background-color: lightgray;
-     padding: 100px 195px 100px 195px;
-     margin: 20px;
- }
-
- .box-4 {
-     background-color: lightgray;
-     padding: 100px 100px 100px 100px;
-     margin: 20px;
- }
-
- .box-5 {
-     background-color: lightgray;
-     padding: 100px 220px 100px 220px;
-     margin: 20px;
- }
-
- .box-6 {
-     background-color: gray;
-     padding: 100px;
+     text-align: center;
      margin: 20px;
  }
 
  .info-box-h1 {
      font-size: 110px;
+     text-align: center;
  }
 
-
- .info-box-row-1 {
-     display: flex;
-     flex-direction: row;
-     text-align: center;
+ .info-sub-row {
+     display: grid;
+     grid-template-columns: 30% 30% 30%;
+     grid-template-rows: 50% 50%;
```

Comment:
See if it works the same without this? Very rarely do I have to set the rows, they usually just work automatically. If this does affect it you can keep, just wanted to check.